articles/rules-are-dangerous.md

```diff
@@ -2,45 +2,45 @@
 authorName: Rolf Bruderer
 authorGithubUsername: bruderol
 issue: 31
-title: Rules are Dangerous
+title: Rules are dangerous
 ---
 # {{page.title}}
 
-Rules followed blindly in a dogmatic way do a lot of harm. This is an uncomfortable truth especially for us as engineers. As such we are used to see the world in rigid rules, true and false, if and else. We like to have crystal clear and exact definitions, guidelines or best practices on how to do something. This is not bad in the first place. But we all must be very careful that we do not fall into the trap of just following such rules blindly without thinking well about consequences. 
+Rules followed blindly in a dogmatic way do a lot of harm. This is an uncomfortable truth especially for us, as engineers. As such we are used to looking at the world in terms of rigid rules, true and false, if and otherwise. We like to have crystal clear and exact definitions, guidelines or best practices on how to do something. This is not a bad thing in itself. But we all must be very careful that we do not fall into the trap of just following such rules blindly without giving careful thought to consequences. 
```

Comment:
Please keep "true and false, if and else" - its a reference to technical programming words as usually used by programmers

Reply:
ok

Reply:
https://github.com/Zuehlke/fifty-shades/commit/36b464815149809af537129d854954e1e795eb1d